src/pages/gen2/build-a-backend/auth/override-cognito/index.mdx

```diff
@@ -27,7 +27,40 @@ export const auth = defineAuth({
 
 You can customize your authentication resource to enable it to behave exactly as needed for your use cases by modifying it directly using [AWS Cloud Development Kit (CDK)](https://aws.amazon.com/cdk/)
 
+## Override Cognito UserPool password policies
+
+You can override the password policy by using the L1 `cfnUserPool` construct and adding a `addPropertyOverride`.
+
 ```ts title="amplify/backend.ts"
+// amplify/backend.ts
+import { defineBackend } from '@aws-amplify/backend';
+import { auth } from './auth/resource.js';
+const backend = defineBackend({
+  auth,
+});
+// extract L1 CfnUserPool resources
+const { cfnUserPool } = backend.resources.auth.resources.cfnResources;
+// use CDK's `addPropertyOverride` to modify properties directly
+cfnUserPool.addPropertyOverride(
+  "Policies",
+  {
+    PasswordPolicy: {
+      MinimumLength: 10,
+      RequireLowercase: true,
+      RequireNumbers: true,
+      RequireSymbols: true,
+      RequireUppercase: true,
+      TemporaryPasswordValidityDays: 20,
+    },
+  }
+);
```

Comment:
```suggestion
);
```
```

Reply:
I'm not sure if that suggestion is going through 😅 but this example is missing its closing backticks

Reply:
OK, I added back the backticks